src/rpcblockchain.cpp

```diff
@@ -399,7 +399,7 @@ UniValue getblock(const UniValue& params, bool fHelp)
             "If verbosity is 1, returns an Object with information about block <hash>.\n"
             "If verbosity is 2, returns an Object with information about block <hash> and information about each transaction. \n"
             "\nArguments:\n"
-            "1. \"blockhash\"          (string, required) The block hash\n"
+            "1. \"hash|height\"          (string, required) The block hash or height\n"
```

Comment:
Move this into the first commit, or into an intermediate commit (so the two together correspond to upstream's commit, and the final commit is our further changes).

Reply:
Done